.circleci/config.yml

```
@@ -9,6 +9,14 @@ workflows:
   build_and_deploy:
     jobs:
       - codacy/checkout_and_version
+      - codacy/sbt:
+          name: check_format
+          cmd: |
+            sbt "set scalafmtUseIvy in ThisBuild := false;
+                 scalafmt::test;
+                 test:scalafmt::test;"
```

Add a check for build files too:
```suggestion
                 sbt:scalafmt::test;
                 test:scalafmt::test;"
```